src/compiler/crystal/command.cr

```diff
@@ -102,7 +102,11 @@ class Crystal::Command
     when File.file?(command)
       run_command(single_file: true)
     else
-      error "unknown command: #{command}"
+      if command =~ /.+#{File::SEPARATOR}.+/ || command.ends_with?(".cr")
```

Comment:
I know I suggested it, but it might be better to leave the separator check out. This implementation may behave unexpected. On windows for example, both `/` and `\` work as separators, so you need'd need to check for either of them.

In any case, the `ends_with?` call should come before the regex because it's faster and should already match in most cases.

Reply:
Also, `command.includes?(File::SEPARATOR)` is simpler and faster. Why do we need a regex?

Reply:
Just `command.ends_with?(".cr")`, please.